frontend/src/metabase/lib/pulse.js

```diff
@@ -136,7 +136,7 @@ export function getActivePulseParameters(pulse, parameters) {
 
       return {
         ...parameter,
-        value: hasParameterValue(pulseParameter)
+        value: hasParameterValue(pulseParameter?.value)
```

Comment:
Not dealing with `getActivePulseParameters` in this PR because I don't know if removing `value: undefined` will have an impact on anything